project/Dependencies.scala

```diff
@@ -46,35 +46,37 @@ object Dependencies {
   }
 
   object Test {
+    private val allTestConfig = "test,it"
 
-    val akkaTypedTestkit = Compile.akkaTypedTestkit % sbt.Test
-    val akkaStreamTestkit = Compile.akkaStreamTestkit % sbt.Test
+    val akkaTypedTestkit = Compile.akkaTypedTestkit % allTestConfig
+    val akkaStreamTestkit = Compile.akkaStreamTestkit % allTestConfig
 
-    val scalatest = "org.scalatest" %% "scalatest" % Versions.scalaTest % sbt.Test
-    val scalatestJUnit = "org.scalatestplus" %% "junit-4-12" % (Versions.scalaTest + ".0") % sbt.Test
-    val junit = "junit" % "junit" % Versions.junit % sbt.Test
+    val scalatest = "org.scalatest" %% "scalatest" % Versions.scalaTest % allTestConfig
+    val scalatestJUnit = "org.scalatestplus" %% "junit-4-12" % (Versions.scalaTest + ".0") % allTestConfig
+    val junit = "junit" % "junit" % Versions.junit % allTestConfig
 
-    val h2Driver = Compile.h2Driver % sbt.Test
-    val postgresDriver = "org.postgresql" % "postgresql" % "42.2.12" % sbt.Test
-    val mysqlDriver = "mysql" % "mysql-connector-java" % "8.0.20" % sbt.Test
-    val msSQLServerDriver = "com.microsoft.sqlserver" % "mssql-jdbc" % "7.4.1.jre8" % sbt.Test
-    val oracleDriver = "com.oracle.ojdbc" % "ojdbc8" % "19.3.0.0" % sbt.Test
+    val h2Driver = Compile.h2Driver % allTestConfig
+    val postgresDriver = "org.postgresql" % "postgresql" % "42.2.12" % allTestConfig
+    val mysqlDriver = "mysql" % "mysql-connector-java" % "8.0.20" % allTestConfig
+    val msSQLServerDriver = "com.microsoft.sqlserver" % "mssql-jdbc" % "7.4.1.jre8" % allTestConfig
+    val oracleDriver = "com.oracle.ojdbc" % "ojdbc8" % "19.3.0.0" % allTestConfig
 
-    val logback = "ch.qos.logback" % "logback-classic" % "1.2.3" % sbt.Test
+    val logback = "ch.qos.logback" % "logback-classic" % "1.2.3" % allTestConfig
 
     val cassandraContainer =
-      "org.testcontainers" % "cassandra" % Versions.testContainers % sbt.Test
+      "org.testcontainers" % "cassandra" % Versions.testContainers % allTestConfig
```

Comment:
not a big issue, but the containers could be `it` only scope. So we don't, by accident add a container test in the wrong place. Though, I don't think we would make that mistake.

Reply:
Agreed.

Reply:
For some reason I can't include the testcontainer modules in `IntegrationTest` exclusively.  They don't get added to the classpath, but I'm not sure why.

Reply:
It may have something to do with extending the `Test` configuration (in order to add test source to the integration classpath).